config/initializers/expire_cache.rb

```diff
@@ -0,0 +1,6 @@
+# During restart, newly enabled or disabled plugins may modify topbar menu. For this reason,
+# cache is invalidated for all users. This will work for the default cache provider (files)
+# but might not work with other providers (memcached). On these installations, users need
```

Comment:
"might not work"... it raises an exception: https://github.com/rails/rails/blob/v4.2.6/activesupport/lib/active_support/cache.rb#L423-L425